File: pgm/ABC_Dart/ABC_364_A.dart
// import 'dart:collection';
import 'dart:io';
// import 'dart:math';
// import 'dart:typed_data';

void main() {
  final int N = int.parse(stdin.readLineSync()!);
  const String Yes = "Yes";
  const String No = "No";
  const String sweet = "sweet";
  const String blank = "";
  const int one = 1;
  const int two = 2;
  const int zero = 0;
  final List<String> S = List<String>.filled(N, blank, growable: false);
  for (int i = zero; i < N; i++) {
    S[i] = stdin.readLineSync()!;
  }
  if (Iterable<int>.generate(N - two).any((i) {
    return S[i] == sweet && S[i + one] == sweet;
  })) {
    print(No);
  } else {
    print(Yes);
  }
  exit(0);
}

File: pgm/ABC_Kotlin/ABC_364_A.kt
// import java.time.*

import kotlin.math.*

const val Yes: String = "Yes"
const val No: String = "No"
const val sweet: String = "sweet"
const val blank: String = ""
const val one: Int = 1
const val zero: Int = 0

fun main() {
    val N: Int = readln().toInt()
    val S: Array<String> = Array(N) { blank }
    for (i in zero until N) {
        S[i] = readln()
    }
    var cnt: Int = zero
    for (i in zero until S.size) {
        if (i != N - 1 && cnt == one && S[i] == sweet) {
            println(No)
            kotlin.system.exitProcess(0)
        } else if (S[i] == sweet) {
            cnt++
        } else {
            cnt = zero
        }
    }
    println(Yes)
    kotlin.system.exitProcess(0)
}